### ISSUE TITLE
Song lists: Series category for featured lists

### ISSUE THREAD

**Issue by Kesera-04** [VocaDB/vocadb#1859]
We have a considerable amount of [lists in the pools section of featured lists for vocaloid series](https://vocadb.net/SongList/Featured?categoryName=Pools&filter=Series%3A&sort=Name). I think they should have their own tab so more people will see them, currently you have to scroll to the bottom and click see more before they show up, and I don't think the Pools category would make anyone think to search for them.
Suggestion illustrated below

![image](https://github.com/user-attachments/assets/ab938899-6223-45e2-a81a-d22c44c55789)

- [✓] Searched **[VocaDB/vocadb](https://github.com/VocaDB/vocadb/issues)** for duplicate issues.

**Comment by andreoda** [comment]
Good idea, we should also think about other categories to add.

For example:
- View lists
- Games
- Series
- Contests

**Comment by Kesera-04** [comment]
Honestly I think your ideas cover most of the content in the pools category, the only things that don't really fit appear to be official playlists, usually made by a streaming service or distribution company. (See [Spotify - KARENT Playlists](https://vocadb.net/SongList/Featured?categoryName=Pools&filter=Spotify&sort=Name) and [ニコニコDAILY TOPICS](https://vocadb.net/SongList/Featured?categoryName=Pools&filter=%E3%83%8B%E3%82%B3%E3%83%8B%E3%82%B3DAILY%20TOPICS&sort=Name)

**Comment by saturclay** [comment]
I think a category for official playlists would be good, and would also cover what I mentioned [here](https://github.com/VocaDB/vocadb/issues/1810#issuecomment-2397983235) about playlists made by artists.

Also, thinking about it again, what if the "Vocaloid rankings" category was just changed to "rankings"? then it wouldn't even need to be different for the other sites.

**Comment by andreoda** [comment]
@Kesera-04
yes, I saw those but was unsure what the category name should be, I guess "Official playlists" is a good name?
@saturclay
yes, it is probably best to rename it to just "Rankings" or "Song rankings".
